pic32/cores/pic32/WInterrupts.c

```diff
@@ -263,12 +265,22 @@ void __attribute__((nomips16,at_vector(_EXTERNAL_0_VECTOR),interrupt(IPL4SRS)))
 void __attribute__((interrupt(),nomips16)) ExtInt0Handler(void)
 #endif
 {
-
+	VEC_ENT_CODE(EXT_INT0,INT0IF,PIN_INT0);
```

Comment:
oh, also, if int retriggers by a new edge, the code doesn't worry about that, setting IFS if it is set does nothing. and if it is too late and retriggers in the epilogue, it will still back-to-back.
Hope that explains it.